exec/jdbc/src/test/java/org/apache/drill/jdbc/test/Drill2489CallsAfterCloseThrowExceptionsTest.java

```diff
@@ -375,6 +375,11 @@ else if (NullPointerException.class == cause.getClass()
 
     public void testAllMethods() {
       for (Method method : jdbcIntf.getMethods()) {
+        if (method.isDefault()) {
```

Comment:
Consider moving to `isOkayNonthrowingMethod()`. What new default methods were introduced in java 10 that require special handling?